src/infrastructure/figma/transformers/figma-node-transformer.ts

```diff
@@ -81,7 +82,7 @@ export const tryTransformNodeToAtlassianDesign = ({
 			: AtlassianDesignStatus.NONE,
 		type: mapNodeTypeToDesignType(node.type),
 		lastUpdated: extra.lastModified,
-		lastUpdatedBy: { id: fileMetaResponse.file.last_touched_by.id },
+		...(lastUpdatedBy ? { lastUpdatedBy: { id: lastUpdatedBy.id } } : {}),
```

Comment:
**suggestion:** Please, see https://github.com/atlassian-labs/figma-for-jira/pull/231/files#r1593215557.